api/controllers/v2/jobs.ts

```diff
@@ -65,20 +65,15 @@ export const HandleJobs = async (event: SQSEvent): Promise<void> => {
         switch (jobStatus) {
           case JobStatus[JobStatus.inQueue]:
             await NotifyBuildProgress(jobId);
-            // start the task , don't start the process before processing the notification
-            const ecsServices = new ECSContainer(c, consoleLogger);
-            const res = await ecsServices.execute(jobId);
-            if (res) {
-              await saveTaskId(jobId, res, consoleLogger);
-            }
-            consoleLogger.info(jobId, JSON.stringify(res));
             break;
           case JobStatus[JobStatus.inProgress]:
             await NotifyBuildProgress(jobId);
             break;
           case JobStatus[JobStatus.timedOut]:
             await NotifyBuildSummary(jobId);
             const taskId = body['taskId'];
+            // for the enhanced application, the taskId will never be defined
+            // as we are not saving it at this time
             if (taskId) {
               await stopECSTask(taskId, consoleLogger);
             }
```

Comment:
completely off topic, but wonder why we don't handle `failed` jobStatus here

Reply:
We do handle the `failed` jobStatus [here](https://github.com/mongodb/docs-worker-pool/blob/086075234892f14a2ccc07bd003b29b03404bbad/api/controllers/v2/jobs.ts#L81)

Reply:
For the failed case, it'll still execute the `NotifyBuildSummary` and `SubmitArchiveJob` I believe